ambry-tools/src/main/java/com.github.ambry/tools/admin/ConcurrencyTestTool.java

```diff
@@ -430,14 +430,13 @@ public void onCompletion(Void result, Exception exception) {
             deleteHelper.deleteBlobAndValidate(blobIdStr, deleteHelper.getErrorCodeForNoError(), callback);
           }
         } catch (InterruptedException e) {
-          logger.error(" Interrupted Exception thrown in , exception ",  e);
+          logger.error(" Interrupted Exception thrown in , exception ", e);
```

Comment:
not your change but can you also fix the error messages in this one and the next? I have done it in #509, but since this is going to go in first, i'll discard those changes from mine